### DISCUSSION TITLE
How to restore journal file after crash

### DISCUSSION THREAD

**Discussion by rik1301** [sqlitebrowser/sqlitebrowser#3822] [Q&A]
Hi, I have suffered a BSOD.  After the crash, I have journal files auto-generated in my files folder, which presumably contains all the edits I've done since opening the program and not done a save.  How do I restore the data contained within these back into the program when I open it please?  I note from previous discussions, if you make an edit to the database then the journal file is automatically deleted and you lose that saved data, so I want to make sure this doesn't happen this time.  Thanks.

**Comment by rik1301** [comment]
:(

**Comment by mgrojo** [comment]
Does this help? https://www.sqlite.org/recovery.html
